.github/ISSUE_TEMPLATE/team_issues.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name: Team issue template
```

Comment:
This issue is only for the xk6-browser team. It won't show up on the new issue screen. IDK how we can use it, though :) I suggest checking it out once we merge this PR and amending it later on if it doesn't work.